src/games/strategy/engine/framework/GameRunner.java

```diff
@@ -635,13 +445,7 @@ public static boolean getCasualtySelectionSlow() {
   private static boolean s_checkedCasualtySelectionSlowPreference = false;
 
   public static void setCasualtySelectionSlow(final boolean casualtySelectionBeta) {
-    final Preferences pref = Preferences.userNodeForPackage(GameRunner.class);
-    pref.putBoolean(CASUALTY_SELECTION_SLOW, casualtySelectionBeta);
-    try {
-      pref.sync();
-    } catch (final BackingStoreException e) {
-      ClientLogger.logQuietly(e);
-    }
+    SystemPreferences.put(SystemPreferenceKey.CASUALTY_SELECTION_SLOW, casualtySelectionBeta);
```

Comment:
SystemPreferences replaces the code chunk above. We get the sync() and error handling for free, and we get type safety with an enum parameter instead of a string. The default value `casualtySelectionBeta` should stay the same, and fewer lines. There were a number of places where this lib was used.